test/unit/service/sharding-table-service.test.js

```diff
@@ -63,4 +63,79 @@ describe('Sharding table service test', async () => {
         );
         expect(bidSuggestion1B).to.be.equal(bidSuggestion1Kb);
     });
+
+    it('Get bid suggestion, option low, returns bid suggestion successfully', async () => {
```

Comment:
Maybe just to add to `it()` above, that is for get bid suggestion without option